handler/src/main/java/io/netty/handler/ssl/Java8SslUtils.java

```diff
@@ -59,4 +62,28 @@ static boolean getUseCipherSuitesOrder(SSLParameters sslParameters) {
     static void setUseCipherSuitesOrder(SSLParameters sslParameters, boolean useOrder) {
         sslParameters.setUseCipherSuitesOrder(useOrder);
     }
+
+    @SuppressWarnings("unchecked")
+    static void setSNIMatchers(SSLParameters sslParameters, Collection<?> matchers) {
+        //sslParameters.setSNIMatchers((Collection<SNIMatcher>) matchers);
+    }
+
+    @SuppressWarnings("unchecked")
+    static boolean checkSniHostnameMatch(Collection<?> matchers, String hostname) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "matchers". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)